config/samples/network_v1beta1_netconfig.yaml

```diff
@@ -40,20 +40,20 @@ spec:
       - end: 172.18.0.250
         start: 172.18.0.10
       cidr: 172.18.0.0/24
-      vlan: 30
+      vlan: 21
   - name: StorageMgmt
     subnets:
     - name: subnet1
       allocationRanges:
       - end: 172.19.0.250
         start: 172.19.0.10
       cidr: 172.19.0.0/24
-      vlan: 40
+      vlan: 22
```

Comment:
right now tenant has `22` and the `172.19.0.0/24` cidr in install_yamls https://github.com/openstack-k8s-operators/install_yamls/blob/main/scripts/gen-nncp.sh#L91 .